frontend/src/fetchers/fetchProducts.tsx

```diff
@@ -1,11 +1,15 @@
+// axios 
 import axios from "axios";
-import { productsType } from "../components/display-products/Types";
+// types 
+import { productsType } from "../components/display-products/displayProductsInterface";
 
 export const fetchProducts = async (URL: string): Promise<productsType[]> => {
+
     try {
         const { data } = await axios.get(URL);
-        return data;
-    } catch (error) {
-        throw new Error(error as string);
+        return data
+    } catch (error: any) {
```

Comment:
```suggestion
    } catch (error: unknown) {
```

Reply:
Will create a branch for fetchers so ignore this.